packages/components/calendar/src/year-picker.tsx

```diff
@@ -140,6 +141,9 @@ export const YearPicker = forwardRef<YearPickerProps, "div">((props, ref) => {
 
           <Portal {...portalProps}>
             <PopoverContent
+              id={id}
```

Comment:
@cidkumagai 
What is passed to aria-controls ?

Reply:
@nakayan5 
aria-controls is set to the id of the tag with the dialog role.